### DISCUSSION TITLE
Error loading sementic model from server - external tools TB3

### DISCUSSION THREAD

**Discussion by 10xape** [TabularEditor/TabularEditor3#1410] [Data Modeling]
Hi

I have power BI desktop connected to already uploaded and refreshing sementic model in Power BI service.

When I try to open it via extranal tools TB3 I get the attached error, teh data is not  imported, it is direct query get data connection.

Can I still make TB3 with POwer BI desktop when I create model out of model ?

Thanks 
![image](https://github.com/user-attachments/assets/739ac1fb-f1c6-4307-8da0-18255441d172)

Matt

**Comment by otykier** [comment]
Hi Matt,

Assuming you have the Business or Enterprise Edition of Tabular Editor 3 (or you're on the 30 day trial), you can connect TE3 to the semantic model through the XMLA endpoint. However, this requires that the [XMLA endpoint is enabled at the capacity level](https://learn.microsoft.com/en-us/power-bi/enterprise/service-premium-connect-tools#enable-xmla-read-write).

It is not possible to connect TE3 to Power BI Desktop, when you're editing a report in Live Connect mode (since in that case, there is no model locally in Power BI Desktop for TE3 to connect to).

**Reply by 10xape** [reply]
thank you

**Reply by 10xape** [reply]
would the proper workflow for live connect Power BI desktop visual editing to make visuals in PB Desktop and adding measures through model connected TE3 itself ?

**Reply by otykier** [reply]
Yes, visual/report development and model development are two separate things. Tabular Editor is only designed for the latter.
